frame/pablo/src/lib.rs

```diff
@@ -412,6 +450,10 @@ pub mod pallet {
 			let base_amount = Self::get_exchange_value(pool_id, pair.base, quote_amount)?;
 			let base_amount_u: u128 = T::Convert::convert(base_amount);
 
+			// https://uniswap.org/whitepaper.pdf
```

Comment:
I think reference to uniswa-paper is not required here. Please also note that if we want to compute fees differently for `Curve` and `Uniswap` we can specialize that part with a new function like `compute_fee()`. If current code is fine then `Uniswap-v2` 's test related to fees should fail and expected values will be required to be adjusted.

Reply:
I was using this while trying to understand this function if it's the same as what uniswap has, IMO we need to move this function to `stable_swap.rs` as the logic is specific to that.

Reply:
fixed